### PR TITLE
Setup translations

### PR DESCRIPTION
**Pull request by lucienbl**
This should setup the translations in order to translate the app! There might be some missing strings I forgot to add but most of them are done.

### CODE REVIEW

**Comment by lucienbl** [comment]
Seems to be some indentation issues, will run lint and update the pull request!

**Comment by amaurymartiny** [comment]
Thanks, that was fast! I'll have a look tonight.

You can run `yarn lint --fix` to fix most of them

**Comment by lucienbl** [comment]
Thanks! Has been done :+1:

File: App/Screens/Home/SmokeVideo/SmokeVideo.js [hunk 1]
```diff
@@ -16,26 +16,39 @@ export class SmokeVideo extends Component {
...
     } = this.props;
 
-    if (cigarettes <= 1) return { opacity: 0.2 };
-    if (cigarettes < 5) return { opacity: 0.5 };
-    if (cigarettes < 15) return { opacity: 0.7 };
-    return { opacity: 1 };
+    if (cigarettes <= 1) return { backgroundColor: '#FFFFFFCC' };
+    if (cigarettes < 5) return { backgroundColor: '#FFFFFFAA' };
+    if (cigarettes < 15) return { backgroundColor: '#FFFFFF22' };
+    return { backgroundColor: '#FFFFFF00' };
```
Comment (amaurymartiny) [inline App/Screens/Home/SmokeVideo/SmokeVideo.js]: Nice! Did the black background disappear on your device?

**Comment by amaurymartiny** [comment]
@lucienbl Could you actually create another new PR with commit 5d2f0a0? Each PR should have only one purpose

File: App/localization/languages/en.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "app_name": "Sh**t! I smoke!",
```
[the remark this answers is not in the document]
Comment (lucienbl) [reply App/localization/languages/en.json]: Oh this string was just the test string I forgot to remove, I think this line can be deleted completely?

[the remark this answers is not in the document]
Comment (amaurymartiny) [reply App/localization/languages/en.json]: Ah yeah, in this case let's remove it

**Comment by lucienbl** [comment]
Sure, github actually added them automatically how can I remove them from the pr? :thinking:

**Comment by amaurymartiny** [comment]
> how can I remove them from the pr? 

Have a look at `git rebase -i HEAD~10`, `drop` the commit, and `git push --force` again. Be sure to backup the dropped commit first.

**Comment by lucienbl** [comment]
Okay got it to create a separate pull request (#70) but to create a "clean" pull request for translations, do I have to rebase a new on and delete this pull request (#69) ? Thanks :)

**Pull request merged**